### ISSUE TITLE
[Bug]: In version v1.1.435, the use of the sai_xl_canny_256lora model not working

### ISSUE THREAD

**Issue by IAn2018cs** [Mikubill/sd-webui-controlnet#2554]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### What happened?

In version v1.1.435, the use of the sai_xl_canny_256lora model with canny is ineffective, but it was effective in version v1.1.434 and earlier versions.

### Steps to reproduce the problem

1. Go to img2img
2. Set args
    * Stable Diffusion checkpoint: dreamshaperXL_turboDpmppSDE.safetensors [676f0d60c8] 
    * Prompt: anime artwork . anime style, key visual, vibrant, studio anime, highly detailed
    * Negative prompt: photo, deformed, black and white, realism, disfigured, low contrast
    * Steps: 6
    * Sampler: DPM++ SDE Karras
    * CFG scale: 2
    * Seed: 2110488101
    * Size: 1024x1408
    * Denoising strength: 0.7
4. Upload img
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/18318073/576d9f5d-73e3-46ca-b633-80a1176c5620)
5. Enable ControlNet canny
<img width="846" alt="image" src="https://github.com/Mikubill/sd-webui-controlnet/assets/18318073/5e1301ce-5c7b-4b54-9be2-f2d663a04735">

### What should have happened?

# v1.1.435 result
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/18318073/a1f1e171-211d-4c0b-a073-742f950f5a8a)

# v1.1.434 result
![image](https://github.com/Mikubill/sd-webui-controlnet/assets/18318073/6460657a-817d-4130-845c-b068e41cc416)

### Commit where the problem happens

webui: 94d4b3c8e78df41eea431069475e1d35f75d02ff
controlnet: 7fb40bf803d05cde9ca9db8b1c6b8ee3c5efb352

### Command Line Arguments

```Shell
webui.sh --api --listen  --port 7860 --xformers
```

### List of enabled extensions

<img width="1059" alt="image" src="https://github.com/Mikubill/sd-webui-controlnet/assets/18318073/7764763f-f08a-4299-8f0f-37792312796b">

### Console logs

```Shell
# v1.1.435
2024-01-23 16:18:53,033 - ControlNet - INFO - unit_separate = False, style_align = False
2024-01-23 16:18:53,033 - ControlNet - INFO - Loading model from cache: sai_xl_canny_256lora [566f20af]
2024-01-23 16:18:53,067 - ControlNet - INFO - Using preprocessor: canny
2024-01-23 16:18:53,067 - ControlNet - INFO - preprocessor resolution = 1030
2024-01-23 16:18:53,103 - ControlNet - INFO - ControlNet Hooked - Time = 0.07777953147888184
100%|██████████| 5/5 [00:01<00:00,  2.87it/s]
Total progress: 100%|██████████| 5/5 [00:02<00:00,  1.88it/s]
{"prompt": "anime artwork  . anime style, key visual, vibrant, studio anime, highly detailed", "all_prompts": ["anime artwork  . anime style, key visual, vibrant, studio anime, highly detailed"], "negative_prompt": "photo, deformed, black and white, realism, disfigured, low contrast", "all_negative_prompts": ["photo, deformed, black and white, realism, disfigured, low contrast"], "seed": 2110488101, "all_seeds": [2110488101], "subseed": 828244603, "all_subseeds": [828244603], "subseed_strength": 0, "width": 1024, "height": 1408, "sampler_name": "DPM++ SDE Karras", "cfg_scale": 2, "steps": 6, "batch_size": 1, "restore_faces": false, "face_restoration_model": null, "sd_model_name": "dreamshaperXL_turboDpmppSDE", "sd_model_hash": "676f0d60c8", "sd_vae_name": null, "sd_vae_hash": null, "seed_resize_from_w": -1, "seed_resize_from_h": -1, "denoising_strength": 0.7, "extra_generation_params": {"ControlNet 0": "Module: canny, Model: sai_xl_canny_256lora [566f20af], Weight: 1, Resize Mode: Crop and Resize, Low Vram: False, Processor Res: 512, Threshold A: 100, Threshold B: 200, Guidance Start: 0, Guidance End: 1, Pixel Perfect: True, Control Mode: Balanced, Hr Option: Both, Save Detected Map: True", "Denoising strength": 0.7}, "index_of_first_image": 0, "infotexts": ["anime artwork  . anime style, key visual, vibrant, studio anime, highly detailed\nNegative prompt: photo, deformed, black and white, realism, disfigured, low contrast\nSteps: 6, Sampler: DPM++ SDE Karras, CFG scale: 2, Seed: 2110488101, Size: 1024x1408, Model hash: 676f0d60c8, Model: dreamshaperXL_turboDpmppSDE, FP8 weight: Enable, Cache FP16 weight for LoRA: False, Denoising strength: 0.7, ControlNet 0: \"Module: canny, Model: sai_xl_canny_256lora [566f20af], Weight: 1, Resize Mode: Crop and Resize, Low Vram: False, Processor Res: 512, Threshold A: 100, Threshold B: 200, Guidance Start: 0, Guidance End: 1, Pixel Perfect: True, Control Mode: Balanced, Hr Option: Both, Save Detected Map: True\", Version: v1.7.0-408-g94d4b3c8"], "styles": ["SAI Anime"], "job_timestamp": "20240123161853", "clip_skip": 1, "is_using_inpainting_conditioning": false, "version": "v1.7.0-408-g94d4b3c8"}

# v1.1.434
2024-01-23 16:29:49,212 - ControlNet - INFO - unit_separate = False, style_align = False
2024-01-23 16:29:49,576 - ControlNet - INFO - Loading model: sai_xl_canny_256lora [566f20af]
2024-01-23 16:29:49,680 - ControlNet - INFO - Loaded state_dict from [/home/amber/workspace/zongai/stable-diffusion-webui/extensions/sd-webui-controlnet/models/sai_xl_canny_256lora.safetensors]
2024-01-23 16:29:49,680 - ControlNet - INFO - Using ControlNet lora (SDXL)
2024-01-23 16:29:49,707 - ControlNet - INFO - ControlNet model sai_xl_canny_256lora [566f20af] loaded.
2024-01-23 16:29:49,730 - ControlNet - INFO - Loading preprocessor: canny
2024-01-23 16:29:49,730 - ControlNet - INFO - preprocessor resolution = 512
2024-01-23 16:29:49,879 - ControlNet - INFO - ControlNet Hooked - Time = 0.6745016574859619
100%|██████████| 5/5 [00:03<00:00,  1.64it/s]
Total progress: 100%|██████████| 5/5 [00:03<00:00,  1.41it/s]
{"prompt": "anime artwork  . anime style, key visual, vibrant, studio anime, highly detailed", "all_prompts": ["anime artwork  . anime style, key visual, vibrant, studio anime, highly detailed"], "negative_prompt": "photo, deformed, black and white, realism, disfigured, low contrast", "all_negative_prompts": ["photo, deformed, black and white, realism, disfigured, low contrast"], "seed": 2110488101, "all_seeds": [2110488101], "subseed": 918257964, "all_subseeds": [918257964], "subseed_strength": 0, "width": 1024, "height": 1408, "sampler_name": "DPM++ SDE Karras", "cfg_scale": 2, "steps": 6, "batch_size": 1, "restore_faces": false, "face_restoration_model": null, "sd_model_name": "dreamshaperXL_turboDpmppSDE", "sd_model_hash": "676f0d60c8", "sd_vae_name": null, "sd_vae_hash": null, "seed_resize_from_w": -1, "seed_resize_from_h": -1, "denoising_strength": 0.7, "extra_generation_params": {"ControlNet 0": "Module: canny, Model: sai_xl_canny_256lora [566f20af], Weight: 1, Resize Mode: Crop and Resize, Low Vram: False, Processor Res: 512, Threshold A: 100, Threshold B: 200, Guidance Start: 0, Guidance End: 1, Pixel Perfect: False, Control Mode: Balanced, Hr Option: Both, Save Detected Map: True", "Denoising strength": 0.7}, "index_of_first_image": 0, "infotexts": ["anime artwork  . anime style, key visual, vibrant, studio anime, highly detailed\nNegative prompt: photo, deformed, black and white, realism, disfigured, low contrast\nSteps: 6, Sampler: DPM++ SDE Karras, CFG scale: 2, Seed: 2110488101, Size: 1024x1408, Model hash: 676f0d60c8, Model: dreamshaperXL_turboDpmppSDE, FP8 weight: Enable, Cache FP16 weight for LoRA: False, Denoising strength: 0.7, ControlNet 0: \"Module: canny, Model: sai_xl_canny_256lora [566f20af], Weight: 1, Resize Mode: Crop and Resize, Low Vram: False, Processor Res: 512, Threshold A: 100, Threshold B: 200, Guidance Start: 0, Guidance End: 1, Pixel Perfect: False, Control Mode: Balanced, Hr Option: Both, Save Detected Map: True\", Version: v1.7.0-408-g94d4b3c8"], "styles": ["SAI Anime"], "job_timestamp": "20240123162949", "clip_skip": 1, "is_using_inpainting_conditioning": false, "version": "v1.7.0-408-g94d4b3c8"}
```

**Comment by huchenlei** [comment]
Update:
Bisect found that #2536 is the culprit. Will investigate why.
